--- libgksu-2.0.12/Makefile.in.orig	2013-03-29 08:45:41.686781605 +0700
+++ libgksu-2.0.12/Makefile.in	2013-03-29 08:45:49.090675796 +0700
@@ -729,7 +729,7 @@
 @INTLTOOL_SCHEMAS_RULE@
 
 @GCONF_SCHEMAS_INSTALL_TRUE@install-data-local:
-@GCONF_SCHEMAS_INSTALL_TRUE@        if test -z "$(DESTDIR)" ; then \
+@GCONF_SCHEMAS_INSTALL_TRUE@	if test -z "$(DESTDIR)" ; then \
 @GCONF_SCHEMAS_INSTALL_TRUE@          for p in $(schemas_DATA) ; do \
 @GCONF_SCHEMAS_INSTALL_TRUE@            GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$$p ; \
 @GCONF_SCHEMAS_INSTALL_TRUE@          done \
