--- mate-panel/panel-run-dialog.c	2013-07-13 01:52:29.000000000 +0700
+++ mate-panel/panel-run-dialog.c.orig	2013-11-03 17:24:25.957806537 +0700
@@ -43,7 +43,7 @@
 #include <matemenu-tree.h>
 
 #define MATE_DESKTOP_USE_UNSTABLE_API
-#include <libmate/mate-desktop-utils.h>
+#include <libmate-desktop/mate-desktop-utils.h>
 
 #include <libpanel-util/panel-error.h>
 #include <libpanel-util/panel-glib.h>
--- applets/clock/calendar-window.c.orig	2013-11-03 17:27:36.248872755 +0700
+++ applets/clock/calendar-window.c	2013-11-03 17:27:47.739070552 +0700
@@ -33,7 +33,7 @@
 #include <gio/gio.h>
 
 #define MATE_DESKTOP_USE_UNSTABLE_API
-#include <libmate/mate-desktop-utils.h>
+#include <libmate-desktop/mate-desktop-utils.h>
 
 #include "calendar-window.h"
 
