--- mate-utils-1.22.1/configure.ac.old	2019-04-24 03:09:08.151268279 +0700
+++ mate-utils-1.22.1/configure.ac	2019-04-24 03:09:21.538393496 +0700
@@ -34,10 +34,6 @@
 if test x$RSVG_CONVERT = x ; then
     AC_MSG_ERROR([could not locate rsvg-convert])
 fi
-AC_PATH_PROG(INKSCAPE, inkscape)
-if test x$INKSCAPE = x ; then
-    AC_MSG_ERROR([could not locate inkscape])
-fi
 
 MATE_COMPILE_WARNINGS([yes])
 
